### PR TITLE
[feat][refactor][chore] Complete Wishlist page

### PR DESCRIPTION
**Pull request by Hamdrive**
Created the page for Wishlist for FueledUp™

Deployed here: [FueledUp™ Wishlist page](https://deploy-preview-9--fueledup.netlify.app/pages/wishlist.html)

Features:
1. Use of grid to dynamically change spacing and layout depending on screen size and content
2. Changed content of cards to remove brief about the product
3. Adapted utility classes to override implementations in grid for wishlist, to prevent other components from breaking
4. Updated footer links to directly visit other pages of FueledUp™
5. Updated navabr to reflect logged in state

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Very clean code, nice work
Good to merge

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        </div>
+      </div>
+      <div class="main my-2">
+        <main class="max-width-1200 mx-auto">
+          <h2 class="page-title txt-bold mx-auto py-md">My Wishlist</h2>
+          <hr />
+          <section
+            class="page-content grid grid-cols-auto grid-col-1 grid-cols-2 gap-1 mx-auto px-lg"
+          >
+            <div class="card mx-auto">
```
Comment (lannisterrr) [inline pages/wishlist.html:93]: would suggest using semantic for the card layout if  you are not using the component library cards and made this specifically for ecom screens.

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+                <div class="flex-row align-center">
+                  <div class="dis-flex flex-wrap align-center">
+                    <p class="txt-md txt-bold mr-sm">₹3,099/-</p>
+                    <s class="strike-color txt-semibold">₹3,400/-</s>
+                  </div>
+                </div>
+              </div>
+              <hr />
+              <div class="card-btns">
+                <button class="btn btn-wish btn-md txt-bold txt-reg w-100">
```
Comment (lannisterrr) [inline pages/wishlist.html:113]: You can also add a button or a badge , so that user can remove the item from the wishlist.

**Review by lannisterrr — COMMENTED** [review]
Very clean UI and other than points mentioned above , the code seems fine.

File: pages/wishlist.html (hunk 1 above)
Comment (Hamdrive) [reply pages/wishlist.html:93]: I have used it from component library, have tweaked it where specifically needed for ecommerce
Overall code does follow semantic practices to the best of my knowledge

File: pages/wishlist.html (hunk 2 above)
Comment (Hamdrive) [reply pages/wishlist.html:113]: Great suggestion, not sure how I forgot about it!
Will add it in the next commit. Thanks for heads up!

**Review by lannisterrr — APPROVED** [review]

**Comment by Hamdrive** [comment]
Code has been reviewed, suggestion given for close btn feature has been added
Closing branch and merging to dev

**Pull request merged**